charts/athens-proxy/README.md.gotmpl

```diff
@@ -0,0 +1,96 @@
+{{- /* run `helm-docs` to generate README.md */ -}}
```

Comment:
Damn, there was a template :smile: Apply comments above on template instead then, sorry about that

Reply:
I would not go so far as to adjust the template to generate a "human readable" version, that would just lead to more ppl. possibly making changes to the readme itself.

The generation of the chart values table is an effort to improve documentation. I am happy to get your opinion on that and if you think it is the right approach.